scripts/tokenomics-emulation/config.json

```diff
@@ -1,9 +1,12 @@
 {
   "network": {
     "url": "wss://rpc.testnet.dev.cere.network:9945",
+    "decimals": 15,
+    "ddc_sc_address": "5FE4RyFK5NuK5GSkebp23fgJNGCg1imCsYTGjLQBQPybJQEU",
+    "gas_limit": -1,
+    "token_amount": 0,
```

Comment:
Let's rename to `smart_contract_cere_token_amount_default`.

Reply:
done